developers.diem.com/docs/wallet-app/try-wallet-admin.md

```diff
@@ -35,11 +52,13 @@ The main page of the admin UI shows the total number of registered users and the
 
 Both these pages offer a list of users. Clicking “Block” button will force the user to be logged out immediately and prevent them from entering the system again.
 
-On the administrators management page “Add administrator” button allows adding a new admin user to the system.
+On the administrators management page, the “Add administrator” button allows adding a new admin user to the system.
```

Comment:
apostrophe?

administrator's management page

Reply:
This is a page for multiple admins, so no possessive apostrophe here.